wiki/Announcements/osu!talk/en.md

```diff
@@ -1,6 +1,6 @@
 # osu!talk
 
-![osu!talk logo](osu!talk.png)
+![osu!talk logo](osu!talk.jpg)
```

Comment:
~~what other files-~~ oh, I guess you mean the translations.. whoops, will check and fix